numpy/f2py/tests/util.py

```diff
@@ -354,3 +357,21 @@ def setup(self):
             self.module = build_module(self.sources, options=self.options,
                                        skip=self.skip, only=self.only,
                                        module_name=self.module_name)
+
+#
+# Helper functions
+#
+
+def getpath(*a):
+    # Package root
+    d = Path(__file__).parent.parent.resolve()
```

Comment:
This may be a bit fragile is files are moved around, but there doesn't seem to be a simple alternative.

Reply:
Agreed, but unless we start providing helpers to return every subproject path maybe something like `np.f2py.get_root()` along the lines of `np.get_include()` I'm not sure we can do better.

Reply:
Though actually I kinda like that idea if anyone cares for it I'm happy to implement it.

Reply:
A solution could be:
```suggestion
    d = Path(numpy.f2py.__file__).parent.resolve()
```
Re: `numpy.f2py.get_root()` - what would be it's use (other than here)? It would be equivalent to `os.path.dirname(numpy.f2py.__file__)` which would not be too bad to be used internally, IMO.

Reply:
Maybe at some point along the way.